lib/nexmo_developer/app/webpacker/stylesheets/application.scss

```diff
@@ -59,4 +59,6 @@
 @import "./layout/search";
 @import "./layout/print";
 
+@import "./active_admin";
```

Comment:
this isn't necessary, `active_admin` has its own pack which is included by the engine